scripts/pylib/twister/twisterlib/reports.py

```diff
@@ -601,6 +601,8 @@ def summary(self, results, ignore_unrecognized_sections, duration):
         )
 
         total_platforms = len(self.platforms)
+        self.filtered_platforms = set(instance.platform.name for instance in self.instances.values()
```

Comment:
it is now just a local variable at `summary()`, isn't it ?
```suggestion
        filtered_platforms = set(instance.platform.name for instance in self.instances.values()
```
and also where it is used below